packages/pages/src/components/address/types.ts

```diff
@@ -40,3 +40,24 @@ export type AddressLineProps = {
   line: AddressLine;
   separator?: string;
 };
+
+/**
+ * The available listing publishers
+ */
+export enum ListingPublisher {
+  googlemybusiness = "googlemybusiness",
+}
+
+/**
+ * The type definition for a Listing
+ */
+export interface ListingType {
+  listingUrl: string;
+  publisher: ListingPublisher;
+}
+
+export enum MapProvider {
```

Comment:
You should do the trick where we allow enum or string so that a user doesn't need to import the type. You can see other examples in other components, such as Image.